src/app/shared/input/check/check.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-checkbox [(ngModel)]="value" (change)="onChange(value)" (blur)="onTouched()" [disabled]="disabled">
```

Comment:
The checkbox is bound to "value" so there should be no need to send "value" onChange?